spec/controllers/users_controller_spec.rb

```diff
@@ -4,6 +4,15 @@
 
 RSpec.describe Api::V1::UsersController, type: :controller do
   let(:user) { create(:user) }
+  let(:admin_role) { create(:role) }
```

Comment:
Can we call this something other than admin?
For example `users_manager_role`?

Reply:
IMO qualifying an admin as someone with ALL the permissions makes sense.

Then, in negative specs, if you want to test something if the admin does not have a certain permission, you remove it manually.

I've been refactoring those for the past two days and IMO this is the cleaner way to do this.

Reply:
I have yet to see a case where we need to discriminate between the admin permissions, for example a case where we need to test if an admin has a certain permission but does not have a certain other.